NAS2D/Math/Rectangle.h

```diff
@@ -91,6 +91,30 @@ namespace NAS2D
 			return Create(startPoint() + offsetAmount, size());
 		}
 
+		constexpr Rectangle operator+(Vector<BaseType> translation) const
+		{
+			return Create(startPoint() + translation, size());
+		}
+
+		constexpr Rectangle operator-(Vector<BaseType> antiTranslation) const
+		{
+			return Create(startPoint() - antiTranslation, size());
```

Comment:
'antiTranslation' is somewhat unexpected naming here but I'm not going split hairs over it. Just figured I'd note it.

Reply:
Yeah, that stuck out as odd to me too. It's still a translation, just in the opposite direction. The direction is implied by the sign. Putting it into the name too sounds a bit like a double negative.

Reply:
It was an attempt to prevent confusion. By naming it `antiTranslation` it makes it clear that the offset will go in the opposite direction, i.e. `A + -B`, a positive number will be negative and a negative number will be positive.

Reply:
Naming it `antiTranslation` make it sound like it's already `-B`, so it sounds like you're doing `A - (-B)`, which would be `A + B`.

Reply:
I would recommend keeping the same term `translation` across all methods.